assets/home/featured-product-sensei-pro.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
I'm considering moving this to the `sections/` folder, to maintain consistency with the other sections. Not sure, though.

Reply:
I can see it either way. Happy with whatever you choose.

Reply:
Hey, decided to move the file in 2219991d70dc5b16d7d1b1ce1467ec88804b1d24 - Thanks! =)